test/transcoding/AtomicFAddEXT.ll

```diff
@@ -20,29 +20,30 @@ define dso_local spir_func float @_Z14AtomicFloatIncRf(float addrspace(4)* align
 entry:
   %0 = addrspacecast float addrspace(4)* %Arg to float addrspace(1)*
   ; CHECK-SPIRV: 7 AtomicFAddEXT [[TYPE_FLOAT_32]]
-  ; CHECK-LLVM: call spir_func float @[[FLOAT_FUNC_NAME:_Z21__spirv_AtomicFAddEXT[[:alnum:]]+]]({{.*}})
+  ; CHECK-LLVM: call spir_func float @[[FLOAT_FUNC_NAME:_Z10atomic_add[[:alnum:]]+ff]]({{.*}})
```

Comment:
I suppose we should be able to consume this variant of LLVM IR in OpenCL 1.2 environment. But there are no `atomic_[add|min|max]` builtins with floating point arguments in the OpenCL C 1.2 spec. Probably, we need a corresponding OpenCL extensions for `SPV_EXT_shader_atomic_float_add` and `SPV_EXT_shader_atomic_float_min_max`